zwrap/example_test.go

```diff
@@ -45,3 +47,20 @@ func Example() {
 	// Output:
 	// {"msg":"Encountered 0 errors.","level":"warn","ts":0,"fields":{}}
 }
+
+func Example_sample() {
+	sampledLogger := zwrap.Sample(zap.NewJSON(), time.Second, 1, 100)
+	// Stub the current time in tests.
+	sampledLogger.StubTime()
+
+	sampledLogger.Error("Unusual failure.")
```

Comment:
nit: think we should move this to after the With, so we know that it's printed even if the common failure is being sampled